pkg/cluster/internal/providers/docker/provider.go

```diff
@@ -58,7 +59,14 @@ func (p *Provider) Provision(status *cli.Status, cluster string, cfg *config.Clu
 		return err
 	}
 
-	if err := ensureNetwork(fixedNetworkName); err != nil {
+	// ensure the pre-requesite network exists
+	networkName := fixedNetworkName
+	if n, set := os.LookupEnv("KIND_EXPERIMENTAL_DOCKER_NETWORK"); set {
```

Comment:
optional nit: maybe just `os.getenv()!=""` to also handle deliberate empty string?

Reply:
They should not set the env? 🤔

Reply:
done